src/devices/somfy.c

```diff
@@ -0,0 +1,178 @@
+/** @file
+    Somfy RTS
+
+    Copyright (C) 2020 Matthias Schulz <mschulz@seemoo.tu-darmstadt.de>
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+*/
+
+#include "decoder.h"
+
+/*
+    Protocol description:
```

Comment:
Repeat the first sentence here.
Do not indent.
Use valid markdown, e.g. verbatim blocks indented by 4 spaces.